### PR TITLE
FIX: image performance

### PR DESCRIPTION
**Pull request by eagerterrier**

### CODE REVIEW

File: ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx [hunk 1]
```diff
@@ -144,6 +144,7 @@ const PostContent = ({ contentBlocks }: { contentBlocks: OptimoBlock[] }) => {
         sizes="(min-width: 1008px) 760px, 100vw"
         className="mediaStyles"
         css={styles.bodyMedia}
+        position={[9]}
```
Comment (amoore108) [inline ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx:147]: Is this just to ensure it's not preloaded?

**Review by amoore108 — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx [hunk 2]
```diff
@@ -47,6 +47,9 @@ const MaskedImage = ({
         mediaType={primaryMimeType || undefined}
         fallbackMediaType={fallbackMimeType || undefined}
         sizes="(min-width: 1008px) 660px, 100vw"
+        preload={true}
+        width={800}
+        height={533}
```
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx:52]: Just curious about these sizes - where did we get the numbers from? Are they needed i.e. could we get away with the preload=true only?

File: ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx (hunk 1 above)
Comment (karinathomasbbc) [inline ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx:147]:
```suggestion
```

**Review by karinathomasbbc — APPROVED** [review]

File: ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx (hunk 1 above)
Comment (eagerterrier) [reply ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx:147]: yes - we just want all images to lazy load. The only one we don't want to lazy load is the header image. I could make it clearer somehow?

File: ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx (hunk 2 above)
Comment (eagerterrier) [reply ws-nextjs-app/pages/[service]/live/[id]/Header/MaskedImage/index.tsx:52]: that's the aspect ratio in `style`. It's just put there to stop layout shift

File: ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx (hunk 1 above)
Comment (eagerterrier) [reply ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx:147]: this is to lazy load images. I could put a real position of a block, but, tbh we want all images lazy loaded as with the banner all images are below the fold

Comment (amoore108) [reply ws-nextjs-app/pages/[service]/live/[id]/Post/index.tsx:147]: The API is a little weird for that component, so if it works maybe just leave it as you have here and we can come back to it. 

A bit hard to reason about within the component as it tries to automatically determine preloading and lazyloading when I feel like they should be done at the prop level.

**Pull request merged**